### DISCUSSION TITLE
[Help]: Center slide

### DISCUSSION THREAD

**Discussion by sarussss** [davidjerleke/embla-carousel#920] [Help]
### Summary

Hi @davidjerleke 
How can I center the content when the slider is not enough to drag, but when the slider is enough much it works normally?
I tried with option [align](https://www.embla-carousel.com/api/options/#align) but no success, do you have any suggestions?

<img width="720" alt="image" src="https://github.com/davidjerleke/embla-carousel/assets/15656996/c434a7aa-4b4f-4693-8a96-8b9d46166ba8">

<img width="720" alt="image" src="https://github.com/davidjerleke/embla-carousel/assets/15656996/01e7ba5b-11b8-4650-8e31-f644c359abb6">

### If applicable, which variants of Embla Carousel are relevant to this question?

- [ ] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-fade
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

### CodeSandbox example

https://stackblitz.com/edit/vitejs-vite-jtxkrl?file=index.html,style.css,main.js

**Comment by janstieler** [comment]
Hi,
with flebox you can center your images with `justify-content: center;` on embla-container.
I don't know if embla have a included positibility to check for the min width fro dragging. But you if you know which the min width is you can calculate it with js and set a class who is setting the necessary flexbox attributes.

**Reply by sarussss** [reply]
Hi @janstieler
I tried this before but had some problems, so now I'm trying some other methods, Thank you very much.

**Reply by janstieler** [reply]
Hi,
As you see in my thread that I have now also problems with the correct centering of the slider.
I think there is something I haven't understood in the way the slider works and the css have to be.

**Reply by sarussss** [reply]
Hi,
you can [check it](https://codesandbox.io/s/gsdkkq)
Hope it helps you somehow.

**Reply by davidjerleke** [reply]
Hi @sarussss!
Why did you close this 🙂? Did you solve it? I'm slower to respond during summer so please be patient. Thanks!

**Reply by sarussss** [reply]
Hi @davidjerleke.
I closed because I have found the solution to the problem I am having, so I am closing it so you don't waste your time checking it.
Thanks you.

**Reply by davidjerleke** [reply]
@sarussss I see. Thanks for clarifying! Feel free to share your solution if you want. Regardless, thanks for all your help!

**Reply by sarussss** [reply]
Thanks @davidjerleke 
My solution is just a temporary solution for a specific case and is not optimal so I don't think I should share it.
